Source/WebCore/rendering/svg/legacy/LegacyRenderSVGContainer.cpp

```diff
@@ -154,8 +155,23 @@ void LegacyRenderSVGContainer::addFocusRingRects(Vector<LayoutRect>& rects, cons
 
 void LegacyRenderSVGContainer::updateCachedBoundaries()
 {
-    SVGRenderSupport::computeContainerBoundingBoxes(*this, m_objectBoundingBox, m_objectBoundingBoxValid, m_strokeBoundingBox, m_repaintBoundingBox);
+    SVGRenderSupport::computeContainerBoundingBoxes(*this, m_objectBoundingBox, m_objectBoundingBoxValid, m_repaintBoundingBox);
     SVGRenderSupport::intersectRepaintRectWithResources(*this, m_repaintBoundingBox);
+    m_strokeBoundingBox = FloatRect::nanRect();
+}
+
+FloatRect LegacyRenderSVGContainer::repaintRectInLocalCoordinatesForHitTesting() const
+{
+    FloatRect strokeBoundingBox = this->strokeBoundingBox();
+    SVGRenderSupport::intersectRepaintRectWithResources(*this, strokeBoundingBox);
```

Comment:
But the input isn't a repaint rect, it's the stroke bounds. More confusion.

Reply:
This is exactly the same to what it was doing before (See old `LegacyRenderSVGShape::updateRepaintBoundingBox` code).
This part is keeping what it is done right now.